Source/JavaScriptCore/b3/B3LowerMacros.cpp

```diff
@@ -537,6 +576,24 @@ class LowerMacros {
         m_changed = true;
     }
 
+    Value* emitDivMod(BasicBlock *block, Opcode nonChillOpcode, Value* num, Value* den)
```

Comment:
This function name is not right since this is only for ARMv7.

Reply:
Hmm. There's nothing ARMv7-specific in the body of the function. That said, AFAIK ARMv7 is the only  that needs it or is going to need it (`RISCV64Assembler.h` has instructions for integer div/mod). How about `callDivModHelper`?